packages/website/tailwind.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  content: ["./src/**/*.{html,tsx,ts,js,jsx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [require("@tailwindcss/forms")],
```

Comment:
Is it possible to include the necessary parts of forms in the ``components``?

Reply:
This confuses me immensely. For some reason, the docusaurus webpack loader picks up the postcss config in website, and then looks for a tailwind config and then fails, even though I don't want it to. This config here fixes that error. I might be able to fix that by disabling postcss in docusaurus' internal webpack config, but I feel like this is annoyingly the cleaner solution.